### PR TITLE
fix: get branches up to date with each other

### PR DESCRIPTION
**Pull request by TheGamer1002**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -8,66 +8,95 @@
...
-    if (m_autonomousCommand != null) {
-      m_autonomousCommand.schedule();
+    autonomousCommand = robotContainer.getAutonomousCommand();
+
+    /*
+     * String autoSelected = SmartDashboard.getString("Auto Selector",
+     * "Default"); switch(autoSelected) { case "My Auto": autonomousCommand
+     * = new MyAutoCommand(); break; case "Default Auto": default:
+     * autonomousCommand = new ExampleCommand(); break; }
+     */
```
Comment (NyxAlexandra) [inline src/main/java/frc/robot/Robot.java:66]: unnecessary

**Review by NyxAlexandra — CHANGES_REQUESTED** [review]
remove the 1 unneeded comment then it's good

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (TheGamer1002) [reply src/main/java/frc/robot/Robot.java]: okay

**Review by NyxAlexandra — APPROVED** [review]
gud

**Pull request merged**